src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -189,7 +200,40 @@ public void showThereIsOnlyOneAccount() {
     }
     //@@author Sherlock-YH
     public void showMainAccountSwitched() {
-        System.out.println("Main Account switched");
         printLine();
+        System.out.println("Current Account switched");
+        printLine();
+        System.out.println("Current Account is:");
+        printLine();
+    }
+
+    public void showCurrentAccount(Account acc){
+        System.out.println("Name: " + acc.getAccountName());
+        System.out.println("Balance: $" + acc.getAccountBalance());
+        printLine();
+    }
+
+    public void showWithdrawCancelled() {
+        System.out.println("Withdrawal has been cancelled!");
+    }
+
+    public void showInsufficientArgsEntered() {
+        System.out.println("missing one of the arguments relevant to the entered command, TRY AGAIN!");
+    }
+
+    public void failToMeetSaveGoal() {
+        System.out.println("Withdrawing the requested amount would lead to failing to meet your personal Save goal");
+        System.out.println("Are you sure you want to follow through? (Y/N)");
+    }
+
+    public void showGoal(SaveGoal goal) {
+        printLine();
```

Comment:
Print the line before this function is called, don't printLine() as the first thing inside the function. Standardisation reasons